File: Tutorials/30-Days-of-Code/Day 2 - Operators.cs
using System;
using System.Collections.Generic;
using System.IO;

class Solution {
    static void Main(String[] args) {
        double mealCost;    // The base cost of the meal
        int tipPercent;     // The integer percentage of tip required
        int taxPercent;     // The integer percentage of tax required
        
        mealCost = Convert.ToDouble(Console.ReadLine ());
        tipPercent = Convert.ToInt16 (Console.ReadLine ());
        taxPercent = Convert.ToInt16 (Console.ReadLine ());
        
        Console.WriteLine ("The total meal cost is {0} dollars.", Math.Round (mealCost + 
                                                                              (mealCost * tipPercent / 100) + 
                                                                              (mealCost * taxPercent / 100)));
    }
}

File: Tutorials/30-Days-of-Code/Day 2 - Operators.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {
    double mealCost;    // The base cost of the meal
    int tipPercent;     // The integer percentage of tip required
    int taxPercent;     // The integer percentage of tax required
    
    scanf ("%lf\n%d\n%d", &mealCost, &tipPercent, &taxPercent);
    
    printf ("The total meal cost is %d dollars.", (int) round (mealCost + 
                                                               (mealCost * tipPercent / 100) + 
                                                               (mealCost * taxPercent / 100)));
    return 0;
}
